src/common/SPELLS/DEATH_KNIGHT.js

```diff
@@ -390,39 +395,33 @@ export default {
     name: 'Scourge Strike',
     icon: 'spell_deathknight_scourgestrike',
   },
-  
+
   SUMMON_GARGOYLE: {
     id: 49206,
     name: 'Summon Gargoyle',
     icon: 'ability_deathknight_summongargoyle',
   },
-  
+
   VIRULENT_PLAGUE: {
     id: 191587,
     name: 'Virulent Plague',
     icon: 'ability_creature_disease_02',
-  },  
+  },
 
   UNHOLY_FRENZY_BUFF: {
     id: 207290,
     name: 'Unholy Frenzy',
     icon: 'spell_shadow_unholyfrenzy',
   },
-  
+
   // Unholy Tier Sets
-  // T20 2 Piece and buff
-  UNHOLY_DEATH_KNIGHT_T20_2SET_BONUS: {
-    id: 242064,
-    name: 'T20 2 set Bonus',
+  // T20 2 Piece
+  UNHOLY_DEATH_KNIGHT_T20_2SET_BONUS_BUFF: {
+    id: 246995,
+    name: 'T20 2 Set Bonus',
     icon: 'spell_deathknight_unholypresence',
```

Comment:
Aren't you gonna run into problems with not being able to check if they have the t202p equipped now?

Reply:
thats unholy. i manage the blood.

Reply:
As said on discord, it will have to be fixed to include both unholy t202p bonus and t202p bonus buff, before I can let it go through, otherwise this PR will break unholy

Reply:
Fixed. Wierd. I didnt touch it. No idea how that happened.

Reply:
Newest commit doesn't do anything in SPELLS?

Reply:
I pr'd a change to SPELLS for uh t20 that might've gone through after you started working on this.  Might try fetch+merge from master and see if you get a conflict.